### ISSUE TITLE
Selecting nodes containing specific text using keyboard only results in an error

### ISSUE THREAD

**Issue by mlind** [freeplane/freeplane#1341]
**Describe the bug**
In "search replace* menu, searching the text and subsequently moving to and selecting all search results using keyboard only results in an error.

Tried it on several times with various mind map sizes.

**To Reproduce**
1. `Ctrl + Shift + F` to open "Search replace"
2. Insert a searchable word to the uppermost box
3. Hit `Tab` three times to get to the selected nodes
4. `Ctrl + A` to select all nodes

**Expected behavior**
Selecting all found nodes succeeds and *Goto* button below is active and selecting the nodes when clicked.

**Mind map files**
[freeplane-bug.zip](https://github.com/freeplane/freeplane/files/12354780/freeplane-bug.zip) - use "idea" as a search word, for example

**Screen recording**
[Video](https://github.com/freeplane/freeplane/assets/3583451/324fc70a-e78c-4596-9214-051d66d98fc0)

**Desktop (please complete the following information):**
 - OS: [Linux, macOS]
 - Version [1.11.5]
 - Java Version [11.0.20 for Linux, 15.0.8 for macOS]

**Additional context**

Current work-around:

1. *Select* a *single node* (with a mouse)
2. Hit `Ctrl + A` once more -> then I can apply *Goto*

Freeplane log:

```
STDERR: Exception in thread "AWT-EventQueue-0"
STDERR: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 15
STDERR: at org.freeplane.view.swing.features.time.mindmapmode.nodelist.TableSorter.modelIndex(TableSorter.java:386)
STDERR: at org.freeplane.view.swing.features.time.mindmapmode.nodelist.TableSorter.getValueAt(TableSorter.java:359)
STDERR: at org.freeplane.view.swing.features.time.mindmapmode.nodelist.NodeList.getMindMapNode(NodeList.java:439)
STDERR: at org.freeplane.view.swing.features.time.mindmapmode.nodelist.NodeList$7.valueChanged(NodeList.java:688)
STDERR: at java.desktop/javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:219)
STDERR: at java.desktop/javax.swing.DefaultListSelectionModel.fireValueChanged(DefaultListSelectionModel.java:186)
STDERR: at java.desktop/javax.swing.DefaultListSelectionModel.setValueIsAdjusting(DefaultListSelectionModel.java:723)
STDERR: at java.desktop/javax.swing.JTable.selectAll(JTable.java:2104)
STDERR: at java.desktop/javax.swing.plaf.basic.BasicTableUI$Actions.actionPerformed(BasicTableUI.java:626)
STDERR: at java.desktop/javax.swing.SwingUtilities.notifyAction(SwingUtilities.java:1810)
STDERR: at java.desktop/javax.swing.JComponent.processKeyBinding(JComponent.java:2900)
STDERR: at java.desktop/javax.swing.JTable.processKeyBinding(JTable.java:5264)
STDERR: at java.desktop/javax.swing.JComponent.processKeyBindings(JComponent.java:2962)
STDERR: at java.desktop/javax.swing.JComponent.processKeyEvent(JComponent.java:2862)
STDERR: at org.freeplane.view.swing.features.time.mindmapmode.nodelist.NodeList$FlatNodeTable.processKeyEvent(NodeList.java:259)
STDERR: at java.desktop/java.awt.Component.processEvent(Component.java:6412)
STDERR: at java.desktop/java.awt.Container.processEvent(Container.java:2263)
STDERR: at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5011)
STDERR: at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
STDERR: at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
STDERR: at java.desktop/java.awt.KeyboardFocusManager.redispatchEvent(KeyboardFocusManager.java:1950)
STDERR: at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchKeyEvent(DefaultKeyboardFocusManager.java:870)
STDERR: at java.desktop/java.awt.DefaultKeyboardFocusManager.preDispatchKeyEvent(DefaultKeyboardFocusManager.java:1139)
STDERR: at java.desktop/java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:1009)
STDERR: at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:835)
STDERR: at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4892)
STDERR: at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
STDERR: at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772)
STDERR: at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
STDERR: at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
STDERR: at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
STDERR: at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
STDERR: at java.base/java.security.AccessController.doPrivileged(Native Method)
STDERR: at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
STDERR: at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
STDERR: at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
STDERR: at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
STDERR: at java.base/java.security.AccessController.doPrivileged(Native Method)
STDERR: at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
STDERR: at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
STDERR: at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
STDERR: at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
STDERR: at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
STDERR: at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
STDERR: at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
STDERR: at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by dpolivaev** [comment]
Many thanks for absolutely perfect bug report

**Comment by dpolivaev** [comment]
Fixed in recently released Freeplane 1.11.6

**Comment by mlind** [comment]
> Fixed in recently released Freeplane 1.11.6

It **is** fixed! Thank you, @dpolivaev !
